lib/editor/components/timetable/CalendarSelect.js

```diff
@@ -49,28 +49,36 @@ export default class CalendarSelect extends Component<Props> {
       <span title={title}>
         <Icon type='calendar-o' /> {label}
         {' '}
-        <BsLabel
-          bsStyle={patternTrips ? 'success' : 'default'}
-          title={`Calendar has ${patternTrips} trips for pattern`}>
-          <Icon type='bars' /> {patternTrips}
-        </BsLabel>
-        {/** {' '}
-        <BsLabel
-          title={`Calendar has trips for ${routeCount} routes`}>
-          <Icon type='bus' /> {routeCount}
-        </BsLabel> **/}
-        {' '}
-        <BsLabel
-          title={`Calendar has ${totalTrips} trips for feed`}>
-          <Icon type='building-o' /> {totalTrips}
-        </BsLabel>
+        {/* $FlowFixMe: need to add two types of options */}
+        {option.type !== 'exception-based'
+          ? <>
+            <BsLabel
+              bsStyle={patternTrips ? 'success' : 'default'}
+              title={`Calendar has ${patternTrips} trips for pattern`}>
+              <Icon type='bars' /> {patternTrips}
+            </BsLabel>
+            {/** {' '}
+            <BsLabel
+              title={`Calendar has trips for ${routeCount} routes`}>
+              <Icon type='bus' /> {routeCount}
+            </BsLabel> **/}
+            {' '}
+            <BsLabel
+              title={`Calendar has ${totalTrips} trips for feed`}>
+              <Icon type='building-o' /> {totalTrips}
+            </BsLabel>
+          </>
+          : <>
+            <Icon type='warning' />(Exception based calendar)
```

Comment:
Does this look good?

Reply:
This comment is directed at you. 
<img width="393" alt="image" src="https://github.com/ibi-group/datatools-ui/assets/63798641/825b1e67-31b2-47a1-8739-10f9cbb3b873">
I'm not sure I ever rly thought that would be a permanent look, I'm open to suggestions. I think something should probably identify it as an exception based service in the `CalendarSelect`

Reply:
Could we add a space between the icon and the text?

Reply:
yes, yes we can

Reply:
We can even do some i18n in https://github.com/ibi-group/datatools-ui/pull/999/commits/369f85f306e5682ad8bca8c74fd7da2477586521

Reply:
To be brief, we could say [Name] (Exception)
Or is `Exception based calendar` a commonly used term?

Reply:
I think Exception based calendar is more clear for the user. We can maybe wait for some feedback if people find it cumbersome